src/templates/base/2019/contributors.html

```diff
@@ -456,10 +482,15 @@ <h1 class="title title-lg">
     }
     window.addEventListener('popstate', handleFilterUrl);
     handleFilterUrl();
+});
 
-    contributorsGrid.setAttribute('data-rendered', true);
-})();
+    try {
+      loaded() 
+    } catch(e) { }
```

Comment:
Any interest in logging when this happens to GA?

Reply:
Done. I've been thinking about adding a more [generic error catch](https://stackoverflow.com/questions/36226532/google-analytics-exception-tracking), though would still need this in this one here as it's in a catch.